src/lib/Form/Type/ContentType/DraftContentTypesDeleteType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\ContentType;
+
+use EzSystems\EzPlatformAdminUi\Form\Data\ContentType\DraftContentTypesDeleteData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class DraftContentTypesDeleteType extends AbstractType
```

Comment:
I would go with a `Draft` namespace similar to `EzSystems\EzPlatformAdminUi\Form\Type\Content\Draft` to avoid using `Draft` in the class name.